evap/staff/forms.py

```diff
@@ -181,6 +181,16 @@ def validate_unique(self):
                     name_field = e.params["unique_check"][1]
                     self.add_error(name_field, e)
 
+    def clean_participants(self):
+        participants = self.cleaned_data.get('participants')
+        if self.instance.pk:
+            voters = self.instance.voters.all()
+            removed_voters = [voter.username for voter in voters if voter not in participants]
```

Comment:
`removed_voters = set(voters) - set(participants)`? might be a bit clearer and also isn't quadratic in runtime. if that complicates the string formatting too much, that's probably not worth it.

Reply:
not too much ;)